[2025-02-12T21:55:06.813+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T21:55:06.827+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-01T04:15:00+00:00 [queued]>
[2025-02-12T21:55:06.836+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-01T04:15:00+00:00 [queued]>
[2025-02-12T21:55:06.836+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T21:55:06.850+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-01 04:15:00+00:00
[2025-02-12T21:55:06.856+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1121) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T21:55:06.857+0000] {standard_task_runner.py:72} INFO - Started process 1127 to run task
[2025-02-12T21:55:06.857+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-01T04:15:00+00:00', '--job-id', '804', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpeet49_3q']
[2025-02-12T21:55:06.858+0000] {standard_task_runner.py:105} INFO - Job 804: Subtask fetch_and_store_crypto_data
[2025-02-12T21:55:06.894+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-01T04:15:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T21:55:06.976+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T04:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T04:15:00+00:00'
[2025-02-12T21:55:06.976+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T21:55:07.302+0000] {logging_mixin.py:190} INFO - Données récupérées : {'BTC': {'USD': 97588.72}, 'ETH': {'USD': 2684.15}}
[2025-02-12T21:55:07.308+0000] {logging_mixin.py:190} INFO - Données insérées dans PostgreSQL.
[2025-02-12T21:55:07.312+0000] {logging_mixin.py:190} INFO - Erreur lors de l'upload dans MinIO : S3 operation failed; code: NoSuchBucket, message: The specified bucket does not exist, resource: /crypto-data, request_id: 182394A2A18F4280, host_id: dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8, bucket_name: crypto-data
[2025-02-12T21:55:07.312+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T21:55:07.323+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T21:55:07.324+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-01T04:15:00+00:00, execution_date=20231001T041500, start_date=20250212T215506, end_date=20250212T215507
[2025-02-12T21:55:07.431+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T21:55:07.447+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T21:55:07.450+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
